package.json

```diff
@@ -4,7 +4,9 @@
   "version": "1.0.2",
   "author": "Jeff Dickey @dickeyxxx",
   "bugs": "https://github.com/heroku/cli-engine-config/issues",
-  "dependencies": {},
+  "dependencies": {
+    "netrc-parser": "^2.0.1"
```

Comment:
Originallly had the netrc stuff in here, but yanked it back out. This dependency is now removed.